Packs/Censys/Integrations/CensysV2/README.md

```diff
@@ -0,0 +1,514 @@
+| Censys.Search.services.port | Number | The port the service was reached at. | 
+| Censys.Search.services.service_name | String | The name of the service on the port. This is typically the L7 protocol \(e.g., “HTTP”\); however, in the case that a more specific HTTP-based protocol is found \(e.g., Kubernetes or Prometheus\), the field will show that. This field indicates where protocol-specific data will be located. | 
+| Censys.Search.services.transport_protocol | String | The transport protocol \(known in OSI model as L4\) used to contact this service \(i.e., UDP or TCP\). | 
+| Censys.Search.parsed.fingerprint_sha256 | String | SHA 256 fingerprint. | 
+| Censys.Search.parsed.issuer.organization | Unknown | The organization name. | 
+| Censys.Search.parsed.names | Unknown | Common names for the entity. | 
+| Censys.Search.parsed.subject_dn | String | Distinguished name of the entity that the certificate belongs to. | 
+| Censys.Search.parsed.validity.end | Date | Timestamp of when certificate expires. Timezone is UTC. | 
+| Censys.Search.parsed.validity.start | Date | Timestamp of when certificate is first valid. Timezone is UTC. | 
+| Censys.Search.parsed.issuer_dn | String | Distinguished name of the entity that has signed and issued the certificate. | 
+
+
+#### Command Example
+```!cen-search index=certificates query="parsed.issuer.common_name: \"Let's Encrypt\"" limit=1```
+
+#### Context Example
+```json
+{
+    "Censys": {
+        "Search": {
+            "parsed": {
+                "fingerprint_sha256": "f3ade17dffcadd9532aeb2514f10d66e22941393725aa65366ac286df9b1234",
+                "issuer": {
+                    "organization": [
+                        "Let's Encrypt"
+                    ]
+                },
+                "issuer_dn": "C=US, O=Let's Encrypt, CN=Let's Encrypt Authority X3",
+                "names": [
+                    "*.45g4rg43g4fr3434g.gb.net",
+                    "45g4rg43g4fr3434g.gb.net"
+                ],
+                "subject_dn": "CN=45g4rg43g4fr3434g.gb.net",
+                "validity": {
+                    "end": "2021-01-10T14:46:11Z",
+                    "start": "2020-10-12T14:46:11Z"
+                }
+            }
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Search results for query "parsed.issuer.common_name: "Let's Encrypt""
+>|Issuer|Issuer dn|Names|SHA256|Subject dn|Validity|
+>|---|---|---|---|---|---|
+>| organization: Let's Encrypt | C=US, O=Let's Encrypt, CN=Let's Encrypt Authority X3 | *.45g4rg43g4fr3434g.gb.net,<br/>45g4rg43g4fr3434g.gb.net | f3ade17dffcadd9532aeb2514f10d66e22941393725aa65366ac286df9b442ec | CN=45g4rg43g4fr3434g.gb.net | start: 2020-10-12T14:46:11Z<br/>end: 2021-01-10T14:46:11Z |
+
+
+## Additional Considerations for this version
+* This version supports the new API version from Censys. Therefore, the integration do not support *websites* searches.
```

Comment:
need to improve the wording here:
1. it is not clear that other than `websites`, the rest of the functionalities are bc
2. need to replace `new` with `v2`
please work with @ShirleyDenkberg on that